### PR TITLE
[test] start and stop test for ServerV2

### PR DESCRIPTION
**Pull request by millken**

### CODE REVIEW

File: api/serverV2_test.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	require := require.New(t)
+	cfg := newConfig(t)
+	config.SetEVMNetworkID(1)
+	svr, bfIndexFile, err := createServerV2(cfg, false)
+	require.NoError(err)
+	defer func() {
+		testutil.CleanupPath(t, bfIndexFile)
+	}()
+	require.NoError(svr.Start())
+	time.Sleep(5 * time.Second) //let server have enough time to start.
```
Comment (Liuhaai) [inline api/serverV2_test.go:31]: Is sleep time 5s too long?

Comment (raullenchai) [reply api/serverV2_test.go:31]: Sleep in the test code will cause flakiness. Pls use wait-until pattern

File: api/serverV2_test.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	cfg := newConfig(t)
+	config.SetEVMNetworkID(1)
+	svr, bfIndexFile, err := createServerV2(cfg, false)
+	require.NoError(err)
+	defer func() {
+		testutil.CleanupPath(t, bfIndexFile)
+	}()
+	require.NoError(svr.Start())
+	require.NoError(testutil.WaitUntil(100*time.Millisecond, 2*time.Second, func() (bool, error) {
+		return true, svr.Stop()
```
Comment (Liuhaai) [inline api/serverV2_test.go:32]: Is the condition always `true`?

File: api/serverV2_test.go [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+func TestNewServerV2(t *testing.T) {
+	require := require.New(t)
+	cfg := newConfig(t)
+	config.SetEVMNetworkID(1)
+	svr, bfIndexFile, err := createServerV2(cfg, false)
+	require.NoError(err)
+	defer func() {
+		testutil.CleanupPath(t, bfIndexFile)
+	}()
+	require.NoError(svr.Start())
```
Comment (CoderZhi) [inline api/serverV2_test.go:30]: after calling this line, what have not been started?

Comment (millken) [reply api/serverV2_test.go:30]: go func() {
		if err := svr.grpcServer.Serve(lis); err != nil {
			log.L().Fatal("grpc failed to serve.", zap.Error(err))
		}
	}()

> after calling this line, what have not been started?

File: api/serverV2_test.go (hunk 2 above)
Comment (millken) [reply api/serverV2_test.go:32]: yes

**Pull request closed without merging**